app/styles/app.css

```diff
@@ -5,6 +5,19 @@ html {
 body {
   margin-bottom: 85px;
   overflow-x: hidden;
+  font-size: 1em !important;
```

Comment:
I don't think we necessarily need it now, but it would be great to get to the bottom of why these `important`'s are needed here. Are these bootstrap overrides? Probably fine to leave it for now and discuss whether we can use `ember-css-modules` to help.

Reply:
These were directly copied/pasted from PASS. Should investigate if all of these are necessary.

Reply:
#35